components/brave_rewards/resources/tip_dialog/components/app.style.ts

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+// TODO: Many of the colors used throughout are "palette" colors.
+// Should we use them by name via css custom properties?
```

Comment:
You should be able to specify colors via styled-components themeing and brave-ui. Both for named colors to support light and dark mode (context menu foreground / background etc) - `${p => p.theme.color.contextMenuBackground}`, but also palette colors if we must `p.theme.palette.grey400`

As a random example, see https://github.com/brave/brave-core/blob/8f3fee0fb79c2c8e7bdff48c138f8d9eb0fdca47/components/brave_new_tab_ui/components/default/settings/index.ts